package.json

```diff
@@ -60,7 +60,7 @@
     "inferno-router": "^8.2.3",
     "inferno-server": "^8.2.3",
     "jwt-decode": "^4.0.0",
-    "lemmy-js-client": "0.19.4",
+    "lemmy-js-client": "0.20.0-alpha.1",
```

Comment:
This one is causing the error.

Reply:
I was able to run the app just fine after making the change to `markdown.ts`.